### DISCUSSION TITLE
Discussions Topics for Jan 31, 2022 Engineering Sync

### DISCUSSION THREAD

**Discussion by carlosfelix2** [CDCgov/prime-reportstream#4063] [Show and tell]
1. @carlosfelix2 New LIVD lookup table - merged with supplemental table and in database
2. @carlosfelix2 Ordering of mappers - in review
3. @carlosfelix2 Release notes - could we just make one running file instead of many?

**Comment by RickHawesUSDS** [comment]
1. @RickHawesUSDS New HL7 standards for COVID-19 OTC reporting
